FrontEnd/SignUpPage/src/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -0,0 +1,267 @@
+import { useForm } from 'react-hook-form';
+import styles from './SignUpFormContent.module.css'
+
+function SignUpFormContentComponent() {
+
+    const errorMessageTemplates = {
+        'required': 'Обов’язкове поле',
+        'email': 'invalid email, should contain @ and .xx',
+        'password': 'Пароль не відповідає вимогам',
+        'confirmPassword': "Паролі не збігаються",
+        'checkboxes': 'Будь ласка, оберіть кого ви представляєте'
+    }
+
+    const emailPattern = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+
+    const passwordPattern = /^(?=.*[0-9])(?=.*[a-zA-Z])[a-zA-Z0-9]{8,20}$/;
+
+
+    const {
+        register,unregister,
+        handleSubmit,
+        formState: { isValid, isSubmitted, errors },
+        watch, getValues, setValue
+    } = useForm();
+
+
+    const onSubmit = (event) => {
+        // event.preventDefault();
+        console.log(isValid)
+        if (isValid) {
+            console.log("Successfully submitted: ", isSubmitted)
+            console.log("Values: ", getValues())
+        }
+    }
+
+
+
+    return (<div className={styles['signup-form']}>
+            <form id='signUpForm' className={styles['signup-form__container']} onSubmit={handleSubmit(onSubmit)}
+                  autoComplete='off'
+                  noValidate>
+                {/* Email and Company field */}
+                <div className={styles["signup-form__row"]}>
+                    {/* Email field */}
+                    <div className={styles["signup-form__column"]}>
+                        <div className={styles["signup-form__label"]}>
+                            <label className={styles["signup-form__label--required"]}>*</label>
+                            <label className={styles["signup-form__label--text"]}>Електронна пошта </label>
+                        </div>
+                        <div className={styles["signup-form__field"]}>
+                            <input
+                                className={styles["signup-form__input"]}
+                                placeholder='Електронна пошта'
+                                type='email'
+                                {...register('email', {
+                                    required: errorMessageTemplates['required'],
```

Comment:
nit: more common way accessing object properties is using `.`. For instance `errorMessageTemplates.required`